messaging/src/main/java/org/axonframework/eventhandling/GenericEventMessage.java

```diff
@@ -90,7 +90,6 @@ public static <P> EventMessage<P> asEventMessage(@Nonnull Object event) {
      *
      * @param name    The {@link QualifiedName name} for this {@link EventMessage}.
      * @param payload The payload of type {@code P} for this {@link EventMessage}.
-     * @see #asEventMessage(Object)
```

Comment:
I left is deprecated and will delete in next PR, because many files use this method and will make this PR even bigger.

Reply:
Thanks for the heads up!